src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravelDate/TravelDateSheet.tsx

```diff
@@ -83,11 +116,21 @@ export const TravelDateSheet = forwardRef<ScrollView, Props>(
           <Section>
             <DateInputSectionItem
               value={dateString}
-              onChange={setDate}
+              onChange={onSetDate}
               maximumDate={maximumDate}
             />
             <TimeInputSectionItem value={timeString} onChange={setTime} />
           </Section>
+          {showActivationDateWarning && (
```

Comment:
This one is **not** showing immediately on date change, only when re-mounting the bottom sheet.